### ISSUE TITLE
Prompt Parameter for onLoad: login-required in js adapter

### ISSUE THREAD

**Issue by floriankick** [keycloak/keycloak#28441]
### Description

Currently using the url parameter prompt=login to force keycloak to show the login form is only possibe for the explicit login() function. I prefer onLoad: login-required parameter in the init() function, but there is no parameter prompt allowed.

Would it be possible to add prompt for the init() onLoad combination as well?

**Comment by jonkoops** [comment]
I don't think we want to add this, for the simple reason that the amount of parameters that can be passed to `login()` is quite large, and I don't want to expand the already quite cluttered initialization options. There is nothing wrong with calling the `.login()` function manually, and only using `init()` to get the authentication status.
